lua/metals/config.lua

```diff
@@ -132,6 +132,7 @@ local valid_metals_settings = {
   "showInferredType",
   "superMethodLensesEnabled",
   "testUserInterface",
+  "enableSemanticHighlighting"
```

Comment:
Total nit, I try to keep these in alphabetical order. Are you able to move this up?

Reply:
fixed